repo/packages/S/security/0/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packagingVersion": "2.0",
+  "name": "helloworld",
+  "version": "0.1.0",
+  "website": "https://github.com/mesosphere/dcos-security",
+  "maintainer": "support@mesosphere.io",
+  "description": "DC/OS security cli package.",
+  "tags": [ "mesosphere", "dcos", "security", "service_accounts", "secrets", "organizartion", "cluster", "users", "groups"]
```

Comment:
since `security` is the package name, does it make sense as a tag as well? I remember doing this for k8s and was told that it's redundant to inject the package name here... not sure if that still holds